From 39ed30ad834a43cf403f88158c6db7a96f1bed29 Mon Sep 17 00:00:00 2001
From: Stefan Weil <sw@weilnetz.de>
Date: Fri, 1 Feb 2019 19:47:46 +0100
Subject: Fix build rule for manpages

This is similar to commit 2106cba0a98a90451df835f3ab7b2aaf54826442
which fixed doc/generate_manpages.sh.

Signed-off-by: Stefan Weil <sw@weilnetz.de>
---
 doc/Makefile.am | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index a218aca3..c0eeda84 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -2,9 +2,6 @@
 
 if ASCIIDOC
 
-asciidoc=asciidoc -d manpage
-
-
 man_MANS = \
   combine_lang_model.1 \
   combine_tessdata.1  \
@@ -29,6 +26,8 @@ man_MANS += \
   unicharset.5
 endif
 
+man_xslt = http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl
+
 EXTRA_DIST = $(man_MANS) Doxyfile
 
 .PHONY: html
@@ -36,7 +35,8 @@ EXTRA_DIST = $(man_MANS) Doxyfile
 html: $(patsubst %,%.html,$(man_MANS))
 
 %: %.asc
-	$(asciidoc) -o $@ $<
+	asciidoc -b docbook -d manpage -o - $< | \
+	xsltproc --nonet $(man_xslt) -
 
 %.html: %.asc
 	asciidoc -b html5 -o $@ $<
-- 
cgit v1.2.1

